src/app/core/services/issue.service.ts

```diff
@@ -428,10 +429,14 @@ export class IssueService {
   private createIssueModel(githubIssue: GithubIssue): Observable<Issue> {
     if (this.phaseService.currentPhase === Phase.phaseBugReporting) {
       return of(BaseIssue.createPhaseBugReportingIssue(githubIssue));
+    } if (this.phaseService.currentPhase === Phase.phaseTeamResponse) {
+      this.issueCommentService.getGithubComments(githubIssue.id, this.isIssueReloaded).pipe(
+        map((githubComments: GithubComment[]) => of(BaseIssue.createPhaseTeamResponseIssue(githubIssue, githubComments)))
```

Comment:
A return statement here?

Reply:
wait a minute.. how did it work without the return 😅